### ISSUE TITLE
Request for new feature - sending picture

### ISSUE THREAD

**Issue by sebswed** [NeiroNx/Intercomm#5]
Hi there. 

I'm not sure if I should post this here as it is not an issue. I was thinking... would it be possible to send a picture via the intercom app? It would be really nice in case of an emergency when no mobile signal is available or when traveling.

Kind regards,
Sebastiaan

**Comment by NeiroNx** [comment]
Yes it may possible. if anroid accpting base64 encoded url with content... 
http://en.wikipedia.org/wiki/Data_URI_scheme
Messages accepting any html.
images processing by that code https://github.com/NeiroNx/Intercomm/blob/master/app/src/main/java/com/nxn/intercomm/MainActivity.java#L494
